pkg/controller/pxc/users_without_dp.go

```diff
@@ -391,28 +391,28 @@ func (r *ReconcilePerconaXtraDBCluster) handleReplicationUserWithoutDP(cr *api.P
 		return nil
 	}
 
-	logger.Info(fmt.Sprintf("User %s: password changed, updating user", user.Name))
+	log.Info(fmt.Sprintf("User %s: password changed, updating user", user.Name))
```

Comment:
any reason to keep this formatted string?

Reply:
Fixed